src/SFA.DAS.Commitments.Api.Client/SFA.DAS.Commitments.Api.Client.csproj

```diff
@@ -1,100 +1,29 @@
+    <PackageTags>Esfa;Das;</PackageTags>
+    <PackageReleaseNotes>Added support for .NetStandard2.0</PackageReleaseNotes>
   </PropertyGroup>
+
   <ItemGroup>
-    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory, Version=3.13.8.999, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.3.13.8\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory.Platform, Version=3.13.8.999, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.3.13.8\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="SFA.DAS.Common.Domain, Version=1.1.0.44010, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\SFA.DAS.Common.Domain.1.1.0.44010\lib\net45\SFA.DAS.Common.Domain.dll</HintPath>
-    </Reference>
-    <Reference Include="SFA.DAS.Http, Version=1.0.0.39136, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\SFA.DAS.Http.1.0.0.39136\lib\net45\SFA.DAS.Http.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Interfaces\IStatisticsApi.cs" />
-    <Compile Include="QueryStringHelper.cs" />
-    <Compile Include="HttpCommitmentHelper.cs" />
-    <Compile Include="EmployerCommitmentApi.cs" />
-    <Compile Include="Interfaces\IHttpCommitmentHelper.cs" />
-    <Compile Include="Interfaces\IEmployerCommitmentApi.cs" />
-    <Compile Include="Interfaces\IRelationshipApi.cs" />
-    <Compile Include="Interfaces\IValidationApi.cs" />
-    <Compile Include="ProviderCommitmentsApi.cs" />
-    <Compile Include="Configuration\ICommitmentsApiClientConfiguration.cs" />
-    <Compile Include="Interfaces\IProviderCommitmentsApi.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="RelationshipApi.cs" />
-    <Compile Include="StatisticsApi.cs" />
-    <Compile Include="ValidationApi.cs" />
```

Comment:
I've checked there are no additional orphan files in the project and all looks fine.